### ISSUE TITLE
v1 staging: fails in walk chain because letsencrypt now redirect you

### ISSUE THREAD

**Issue by az143** [lukas2511/dehydrated#489]
i'm testing my setup with the current/v1 staging setup, and dehydrated -c fails in walk chain because
http://cert.stg-int-x1.letsencrypt.org/ is no longer available in that location; instead letsencrypt redirects you to https://.

workaround for me: tell curl to follow redirections, ie add CURL_OPTS="-L" to the dehydrated config.

```
$ dehydrated -e
...
declare -- CA="https://acme-staging.api.letsencrypt.org/directory"
...
$ dehydrated -c
...
+ Creating fullchain.pem...
 + Walking chain...
  + ERROR: An error occurred while sending get-request to http://cert.stg-int-x1.letsencrypt.org/ (Status 301)

Details:
<html>
<head><title>301 Moved Permanently</title></head>
<body bgcolor="white">
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>


ERROR: Walking chain has failed, your certificate has been created and can be found at <hidden>, the corresponding private key at <hidden>.
```

**Comment by lukas2511** [comment]
Are you using the newest version from git? This should already be fixed since 7a0e71c6c2ccc6e98abca5ea1c7de28053e90c02.

**Comment by az143** [comment]
you're right. i'm sorry for the noise; i'm using 0.5.0 and somehow i missed the commit comment about this having been fixed since the release.
